src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private Fruit fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, Fruit fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public FruitTransaction() {
+    }

```

Comment:
Empty constructor is not necessary if you already have a parameterized constructor and you're not using any framework that requires an empty constructor.

Reply:
ok, but I'm using it in DataConverterImpl